eras/shelley-ma/impl/src/Cardano/Ledger/ShelleyMA/TxBody.hs

```diff
@@ -21,20 +21,29 @@ module Cardano.Ledger.ShelleyMA.TxBody
   ( MATxBody
       ( MATxBody,
         TxBodyConstr,
-        TxBody',
-        adHash',
-        certs',
-        inputs',
-        mint',
-        outputs',
-        txfee',
-        update',
-        vldt',
-        wdrls'
+        MATxBody_,
+        matbAuxDataHash,
+        matbCerts,
+        matbInputs,
+        matbMint,
+        matbOutputs,
+        matbTxFee,
+        matbUpdate,
+        matbValidityInterval,
+        matbWdrls,
+        matbAuxDataHash_,
+        matbCerts_,
+        matbInputs_,
+        matbMint_,
+        matbOutputs_,
+        matbTxFee_,
+        matbUpdate_,
+        matbValidityInterval_,
+        matbWdrls_
```

Comment:
Here the set of exports with trailing `_` have been added like the set of exports for identifiers without trailing `_`, owing to the two separate pattern synonyms existing for `MATxBody`

Reply:
Sorry, I totally messed up. This is the only era that provides a special pattern synonym for deconstructing only.

We need to get rid of it and other accessors (like `inputs'`, `outputs'`, etc.) in other eras, but we  probably should do that in a separate PR. I created an issue for this : #3130 
Could you please revert this change of tick to underscore and the prefix as well `certs'` -> `matbCerts_` for this pattern `MATXBody'`/`MATXBody_`. Alternatively you can just get rid of this pattern synonym in this PR if you prefer.

Reply:
I got rid of it!